packages/flutter/test/widget/custom_multi_child_layout_test.dart

```diff
@@ -11,7 +11,8 @@ class TestMultiChildLayoutDelegate extends MultiChildLayoutDelegate {
   BoxConstraints getSizeConstraints;
 
   Size getSize(BoxConstraints constraints) {
-    getSizeConstraints = constraints;
+    if (!RenderObject.debugInDebugDoesMeetConstraints)
```

Comment:
This seems slightly dangerous in that end-developers could easily write code that worked very differently in checked and non-checked mode....  Maybe we should put a note in the exception printout if debugInDebugDoesMeetConstraints is true?  Some sort of hint to help developers figure out what's happening....